apps/editing-toolkit/editing-toolkit-plugin/block-patterns/class-block-patterns-from-api.php

```diff
@@ -32,11 +32,14 @@ class Block_Patterns_From_API {
 	 * Block_Patterns constructor.
 	 */
 	private function __construct() {
+		$this->is_site_editor_enabled = function_exists( 'gutenberg_is_fse_theme' ) && gutenberg_is_fse_theme();
+		// Tells the backend which patterns source site to default to.
+		$this->patterns_source    = $this->is_site_editor_enabled ? 'fse_block_patterns' : 'block_patterns';
```

Comment:
@andrewserong   I was testing loading FSE patterns into the site editor with this PR>

The API response itself (changes in D57962-code) work a treat when called independently, but with everything sandboxed, the ETK plugin still returns a response from the existing `block_patterns` source site.

I can confirm that `$this->is_site_editor_enabled` is `true` for my test FSE site and the request url is something like https://public-api.wordpress.com/rest/v1/ptk/patterns/en?tags=pattern&pattern_meta=is_web&patterns_source=fse_block_patterns

I'm not sure if it's caching or the ETK plugin only fetches from production? Any hints?

Thanks!

Reply:
>  I believe the objective was to use dotcomfsepatterns as the source site for just the site editor itself, but editing pages and posts should still use dotcompatterns.

Following on from a discussion with @andrewserong this PR will need to address the above concern. 👍 

p1615290788048300-slack-CS0SC36SX

Reply:
The `patterns_source` param is working now that D57962-code has landed.

In https://github.com/Automattic/wp-calypso/pull/50694/commits/c4497ba2e0899cd826cbc0b88c85dd0a14171d38 I've updated the `current_screen` check to see if we're in the site editor.